### PR TITLE
Rootstock 2019 03 18

### PR DESCRIPTION
**Pull request by trangdata**
update changes from manubot rootstock

### CODE REVIEW

**Comment by trangdata** [comment]
@dhimmel Could you help review the PR?

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ We encourage users to openly license their manuscripts, which is the default as
 [![License: CC0 1.0](https://img.shields.io/badge/License%20Parts-CC0%201.0-lightgrey.svg)](https://creativecommons.org/publicdomain/zero/1.0/)
 
 Except when noted otherwise, the entirety of this repository is licensed under a CC BY 4.0 License ([`LICENSE.md`](LICENSE.md)), which allows reuse with attribution.
-Please attribute by linking to https://github.com/trang1618/tpot-ds-ms.
```
Comment (dhimmel) [inline README.md]: You will want to keep this line as is.

File: README.md [hunk 2]
```diff
@@ -113,8 +113,4 @@ All other files are only available under CC BY 4.0, including:
 + `*.pdf`
 + `*.docx`
 
-Except for the following files with different licenses:
-
-+ `build/assets/anchors.js` which is [released](https://www.bryanbraun.com/anchorjs/) under an [MIT License](https://opensource.org/licenses/MIT)
-
-Please open [an issue](https://github.com/trang1618/tpot-ds-ms/issues) for any question related to licensing.
+Please open [an issue](https://github.com/manubot/rootstock/issues) for any question related to licensing.
```
Comment (dhimmel) [inline README.md]: Change this URL back to `https://github.com/trang1618/tpot-ds-ms/issues`

File: ci/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Continuous integration / analysis directory
 
-[![Build Status](https://travis-ci.org/greenelab/manubot-rootstock.svg?branch=master)](https://travis-ci.org/greenelab/manubot-rootstock)
+[![Build Status](https://travis-ci.com/manubot/rootstock.svg?branch=master)](https://travis-ci.com/manubot/rootstock)
```
Comment (dhimmel) [inline ci/README.md]: Change this line to 

```
[![Build Status](https://travis-ci.org/trang1618/tpot-ds-ms.svg?branch=master)](https://travis-ci.org/trang1618/tpot-ds-ms)
````

File: ci/deploy.sh [hunk 4]
```diff
@@ -49,8 +49,8 @@ This build is based on
 https://github.com/$TRAVIS_REPO_SLUG/commit/$TRAVIS_COMMIT.
 
 This commit was created by the following Travis CI build and job:
-https://travis-ci.org/$TRAVIS_REPO_SLUG/builds/$TRAVIS_BUILD_ID
-https://travis-ci.org/$TRAVIS_REPO_SLUG/jobs/$TRAVIS_JOB_ID
+https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_BUILD_ID
+https://travis-ci.com/$TRAVIS_REPO_SLUG/jobs/$TRAVIS_JOB_ID
```
Comment (dhimmel) [inline ci/deploy.sh]: Keep as `.org`

**Review by dhimmel — COMMENTED** [review]
Nice job with most of the conflicts.

[The build logs](https://travis-ci.org/trang1618/tpot-ds-ms/builds/507989598#L2077) look good.

I commented on a few sentences that could be changed.

File: README.md (hunk 1 above)
Comment (trangdata) [reply README.md]: I see. I thought this part refers to the Manubot tool itself. Fixed [here](https://github.com/trang1618/tpot-ds-ms/pull/8/commits/fcb528c8a16dfb53e6e79ccefdf1ecccaccfb0c9).

**Review by dhimmel — APPROVED** [review]
From https://github.com/manubot/rootstock/blob/master/SETUP.md#merging-upstream-rootstock-changes

> If updating master via a pull request, proceed to push the commit to GitHub and open a pull request. Once the pull request is ready to merge, use GitHub's "Create a merge commit" option rather than "Squash and merge" or "Rebase and merge" to preserve the rootstock commit hashes.

I think you are ready to merge this PR with "Create a merge commit"

**Pull request merged**